### DISCUSSION TITLE
I've hosted my website on github pages but its not running

### DISCUSSION THREAD

**Discussion by ginakena** [community/community#133809] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Pages

### Body

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages.

[GitHub Status](https://githubstatus.com/) — [@githubstatus](https://twitter.com/githubstatus)

**Comment by davevad93** [comment, marked as answer]
Hi @ginakena ,

The issue with your repo is that the structure isn't correct, you need an `index.html` in the root. Your structure is the following: 

`/webby`
`└── /website`
              `         ├── /Images`
              `         └── index.html`


But the right structure should be this:

`/webby`
`├── /Images`
`└── index.html`

So basically you have to delete the  `website` folder and put its contents in the root of the repo.

I noticed another problem with your image paths, you point to your local folder, for example:

```css
background: url('file:///C:/Users/PC/Desktop/website/Images/Background.jpg')
```

That isn't correct, you should change the path according to the structure of your project, which is:

```css
background: url('Images/Background.jpg')
```

You have to do that with all the paths of your images in the nested css in your `index.html`.

**Reply by ginakena** [reply]
Hello
Alright, let me try and rectify that

**Reply by davevad93** [reply]
👍 @ginakena , When you're done with the changes redeploy your `github pages` environment from the `main` branch, otherwise it will still point to your old `Web` branch and you'll still get the warning "**The site configured at this address does not contain the requested file.**"

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @ginakena ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Sivamani-18** [comment]
It sounds like there are a few potential issues that could be causing your website to not display properly on GitHub Pages. Here are some steps to troubleshoot and resolve the issue:

1. **Check the Repository Settings:**
   - Ensure that GitHub Pages is enabled for your repository. Go to the repository settings, scroll down to the "GitHub Pages" section, and make sure a branch is selected as the source (usually `main` or `gh-pages`).

2. **Ensure the Presence of `index.html`:**
   - Make sure you have an `index.html` file in the root of your repository or in the specified folder if you're using a different branch/folder for GitHub Pages.

3. **Check Case Sensitivity:**
   - GitHub Pages is case-sensitive. Ensure that all file and folder names match exactly (e.g., `index.html` is not the same as `Index.html`).

4. **Verify the File Permissions:**
   - Ensure that the files have the correct permissions set, though this is less common for GitHub Pages issues.

5. **Check the Build Process (if applicable):**
   - If you're using a static site generator (like Jekyll, Hugo, etc.), ensure that the build process is correctly configured and the generated files are in the correct location.

6. **Review GitHub Pages Documentation:**
   - Go through the [GitHub Pages documentation](https://docs.github.com/en/pages) to ensure all configurations are correctly set.

Here’s a detailed checklist you can follow:

### Checklist for GitHub Pages

1. **Repository Settings:**
   - Navigate to your repository on GitHub.
   - Go to "Settings" > "Pages".
   - Ensure the source is set to the correct branch (`main` or `gh-pages`) and folder (usually `/ (root)`).

2. **`index.html` File:**
   - Ensure an `index.html` file is present in the root directory of your repository.
   - If using a different folder or branch, ensure `index.html` is in the correct location.

3. **File Names and Paths:**
   - Double-check that all filenames and paths are correct and match the case (e.g., `index.html` instead of `Index.html`).

4. **Permissions:**
   - While rare for GitHub Pages, ensure that files are readable.

5. **Static Site Generators:**
   - If using Jekyll:
     - Ensure your `_config.yml` is correctly configured.
     - Verify that the `_site` folder contains the generated `index.html`.
   - If using another generator, check its build process and output folder.

6. **Commit and Push:**
   - Make sure all changes are committed and pushed to the repository.

### Example of a Simple Setup

1. **Directory Structure:**
   ```
   my-website/
   ├── index.html
   ├── about.html
   ├── css/
   └── js/
   ```

2. **Basic `index.html` Content:**
   ```html
   <!DOCTYPE html>
   <html lang="en">
   <head>
       <meta charset="UTF-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
       <title>My GitHub Pages Site</title>
   </head>
   <body>
       <h1>Welcome to My Website</h1>
       <p>This is my awesome website hosted on GitHub Pages!</p>
   </body>
   </html>
   ```

If you follow these steps and still encounter issues, please provide more details about your repository structure and any error messages you’re seeing. This will help in diagnosing the problem further.
